public/locales/ru.json

```diff
@@ -1,8 +1,10 @@
 {
-	"404-page": {
-		"content_title": "Упс! Мы не нашли эту страницу",
-		"content_desc": "Но вы можете вернуться на главную страницу",
-		"link_back_to_home": "Вернуться на главную"
+	"products": {
+		"add_to_cart": "добавить в корзину",
```

Comment:
need to be in capital letters

Reply:
I decided to do everything with a small letter here